enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -637,18 +653,26 @@ func (mc *MigrationCache) copyDataInBackground() error {
 	}
 }
 
-func (mc *MigrationCache) copy(c *copyData) {
-	defer c.ctxCancel()
+func (mc *MigrationCache) logCopyChanFullInBackground() error {
```

Comment:
I didn't see an error returned in this function

Reply:
Done